src/heatdesalination/heat_pump.py

```diff
@@ -149,11 +158,14 @@ def _coefficient_of_performance(
 
     """
 
-    return (
-        system_efficiency
-        * (condensation_temperature + pinch_point_temperature_difference)
-        / (condensation_temperature - evaporation_temperature)
-    )
+    _cop_carnot = (
```

Comment:
Adaptation of model to that used by Gangar et al., 2020

Gangar N, Macchietto S, Markides CN (2020) Recovery and Utilization of Low-Grade Waste Heat in the Oil-Refining Industry Using Heat Engines and Heat Pumps: An International Technoeconomic Comparison. Energies 13:2560